ginga/misc/plugins/Cuts.py

```diff
@@ -204,6 +217,15 @@ def build_gui(self, container):
         vbox2.add_widget(Widgets.Label(''), stretch=1)
 
         vbox.add_widget(vbox2, stretch=0)
+
+        # Create axes checkboxes only if file is multidimensional
+        image = self.fitsimage.get_image()
+        if not hasattr(image, 'naxispath'):
+            pass
+        elif len(image.naxispath) > 0:
+            self.mdfile = True
+            self.build_axes(vbox)
+
```

Comment:
`redo()` is called when a new image is loaded into the viewer, so this test needs to be performed there as well, *before* the calls to replot.